resources/testdata/data/json/dates/gregory@1/ar-EG.json

```diff
@@ -123,10 +123,73 @@
       "short": "h:mm a"
     },
     "date_time": {
-      "full": "{1} في {0}",
-      "long": "{1} في {0}",
-      "medium": "{1}, {0}",
-      "short": "{1}, {0}"
+      "style_patterns": {
+        "full": "{1} في {0}",
+        "long": "{1} في {0}",
+        "medium": "{1}, {0}",
+        "short": "{1}, {0}"
+      },
+      "available_formats": {
+        "Bh": "h B",
+        "Bhm": "h:mm B",
+        "Bhms": "h:mm:ss B",
+        "d": "d",
+        "E": "ccc",
+        "EBhm": "E h:mm B",
+        "EBhms": "E h:mm:ss B",
+        "Ed": "E، d",
+        "Ehm": "E h:mm a",
+        "EHm": "E HH:mm",
+        "Ehms": "E h:mm:ss a",
+        "EHms": "E HH:mm:ss",
+        "Gy": "y G",
+        "GyMMM": "MMM y G",
+        "GyMMMd": "d MMM y G",
+        "GyMMMEd": "E، d MMM y G",
+        "h": "h a",
+        "H": "HH",
+        "hm": "h:mm a",
+        "Hm": "HH:mm",
+        "hms": "h:mm:ss a",
+        "Hms": "HH:mm:ss",
+        "hmsv": "h:mm:ss a v",
+        "Hmsv": "HH:mm:ss v",
+        "hmv": "h:mm a v",
+        "Hmv": "HH:mm v",
+        "M": "L",
+        "Md": "d/‏M",
+        "MEd": "E، d/‏M",
+        "MMdd": "dd‏/MM",
+        "MMM": "LLL",
+        "MMMd": "d MMM",
+        "MMMEd": "E، d MMM",
+        "MMMMd": "d MMMM",
+        "MMMMEd": "E، d MMMM",
+        "MMMMW-count-zero": "الأسبوع W من MMMM",
+        "MMMMW-count-one": "الأسبوع W من MMMM",
+        "MMMMW-count-two": "الأسبوع W من MMMM",
+        "MMMMW-count-few": "الأسبوع W من MMMM",
+        "MMMMW-count-many": "الأسبوع W من MMMM",
+        "MMMMW-count-other": "الأسبوع W من MMMM",
```

Comment:
Do you have the plural rule selection implemented?

Reply:
No I don't, and I haven't really thought through this problem. Do you have recommendations here? Currently I discard them in the `Skeleton` struct construction. I don't really have a mental model of how these will be used.

Reply:
Ok, I'm going to try to change the representation here to handle these. I see what's going on with them now.

Reply:
If you don't want to feature-creep your work, open an issue to figure these out later, and in the mean time, copy only the "other" form into the data.